### PR TITLE
more m1 fixes for bobs

### PR DESCRIPTION
**Pull request by SteveRuble**
- moved m1 logic to install step
- Bump version to 0.1.2

### CODE REVIEW

File: lib/bobs/scala/step.sh [hunk 1]
```diff
@@ -0,0 +1,29 @@
+#!/bin/bash
+
+# IH_CORE_DIR will be set to the directory containing the bin and lib directories.
+
+function ih::setup::bobs.scala::help() {
+  echo 'This will install the correct scala
```
Comment (vanzin) [inline lib/bobs/scala/step.sh:6]: Kinda curious whether this is needed. Isn't bazel supposed to handle all scala-related dependencies itself?

File: lib/core/m1/default/01_m1.sh [hunk 2]
```diff
@@ -0,0 +1,15 @@
...
+else
+  eval "$(/usr/local/bin/brew shellenv)"
+fi
+
+# Aliases to switch compatibility mode in shells
+alias x86='arch -x86_64 $SHELL'
+alias amd64='arch -arm64 $SHELL'
+
+alias ibrew='arch -x86_64 /usr/local/bin/brew'
+alias mbrew='arch -arm74 /opt/homebrew/bin/brew'
```
Comment (vanzin) [inline lib/core/m1/default/01_m1.sh:15]: arm64?

Comment (SteveRuble) [reply lib/core/m1/default/01_m1.sh]: yep

File: lib/bobs/scala/step.sh (hunk 1 above)
Comment (SteveRuble) [reply lib/bobs/scala/step.sh:6]: Yeah, this is for the IDE itself, which wants Scala stuff I guess. That's also why Go needs to be installed.

**Review by vanzin — APPROVED** [review]

**Pull request merged**